src/index.js

```diff
@@ -336,6 +336,12 @@ exports.init = async (Implementation) => {
   }
 };
 
+function getSmartActionsUsingHTTPMethod(actions) {
+  return actions.filter(
+    (action) => Object.hasOwnProperty.call(action, 'httpMethod'),
```

Comment:
[Just opinion]
to do this I use simpler `action => action.httpMethod` since I'm sure httpMethod is not defined higher in prototype chain (maybe I'm wrong?)

Reply:
I'm not sure this can happen, but with my syntax, the log will also be displayed with `httpMethod: null` or `httpMethod: undefined`, which would not be the case with your syntax

Reply:
ah! nice catch